packages/framework/src/Component/Router/FriendlyUrl/CompilerPass/FriendlyUrlDataProviderCompilerPass.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
I really like the whole solution :+1: however I would really appreciate it the commit message was much more descriptive than it is now. There should be more points explaining what things exactly changed :slightly_smiling_face: